woof-distro/x86_64/ubuntu/jammy64/DISTRO_PET_REPOS

```diff
@@ -32,10 +32,10 @@ PET_REPOS='
 z|http://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
 z|http://ftp.nluug.nl/ftp/pub/os/Linux/distr/puppylinux|Packages-puppy-*-official
 z|http://ftp.cc.uoc.gr/mirrors/linux/puppylinux|Packages-puppy-*-official
-z|ftp://mirror.aarnet.edu.au/pub/puppylinux|Packages-puppy-*-official
+z|https://mirror.aarnet.edu.au/pub/puppylinux|Packages-puppy-*-official
 z|http://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
 z|http://ftp.nluug.nl/ftp/pub/os/Linux/distr/puppylinux|Packages-puppy-*-official
-z|ftp://ftp.vcu.edu/pub/gnu+linux/puppylinux|Packages-puppy-*-official
+z|https://ftp.vcu.edu/pub/gnu+linux/puppylinux|Packages-puppy-*-official
```

Comment:
Bad certificate

Reply:
But the download from this mirror **will** work because of [woof-CE/woof-code/support/download_file.sh](https://github.com/puppylinux-woof-CE/woof-CE/blob/38e2eaf135598b2a8f48d1e0955f6955f069c566/woof-code/support/download_file.sh#L32) 👎🏿

Reply:
I wonder, should I remove that, or change it back to FTP? My main issue with FTP is its lack of integrity checks. If they are performed internally in woof-CE, using FTP can be... not that bad idea. For example, Debian uses HTTP because all packages are signed and checked after each download. Also, as far as I know, Wine can download some packages through HTTP and then check their checksums. So, if woof-CE checks the integrity of its downloads, then HTTP/FTP/Git Protocol aren't that much of an issue (in that case, I can simply close this pull request).

Reply:
woof-CE doesn't do GPG verification and integrity checks (hashes, etc') are not an option here, because these are not static files that don't change

Reply:
IMO 2-3 HTTPS mirrors in different continents are more than enough, so those that only speak FTP and HTTP can be removed

Reply:
OK, all git:// and ftp:// were switched to https://. Also, I've switched some http:// to https://, and removed all mirrors I was not sure about.

Reply:
Resolving the conversation as the problem mirror has been removed in the pull request.